### DISCUSSION TITLE
请教5.15和5.10内核的系统停止更新了吗？

### DISCUSSION THREAD

**Discussion by Saintlord77** [ophub/amlogic-s9xxx-armbian#2552] [Amlogic]
我看大佬更新的8-9月镜像都是只有6.1和6.6内核的，之前的5.15和5.10的不再更新了吗？

**Comment by ophub** [comment]
一直在更新，可以`armbian-update -k 5.15`进行选择更新
现在可以选择的内核有5.4.y, 5.10.y, 5.15.y, 6.1.y, 6.6.y

https://github.com/ophub/kernel/releases/tag/kernel_stable

**Comment by Saintlord77** [comment]
谢谢大佬，也就是以后不会出打包好的5.15的了对吧。N1的6.1和6.6内核wifi驱动都有问题，我更新一下内核试试。
